src/main/java/org/jenkinsci/plugins/suppress_stack_trace/SuppressionFilter.java

```diff
@@ -85,7 +86,7 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
                     w.println(Messages.SuppressionFilter_Footer());
                     w.println("</body></html>");
                     w.close();
-                } catch (Exception x) {
+                } catch (Throwable x) {
```

Comment:
What happens if there is an `OutOfMemoryError`? you may not be able to create the `ServletException`